app/screens/full-onboarding-flow/full-onboarding-flow.tsx

```diff
@@ -0,0 +1,168 @@
+  const accountId = dataDebug?.me?.defaultAccount?.id
+
+  const [onboardingFlowStart] = useOnboardingFlowStartMutation()
+
+  const [firstName, setFirstName] = useState("")
+  const [lastName, setLastName] = useState("")
+
+  const confirmNames = async () => {
+    Alert.alert(
+      LL.FullOnboarding.confirmNameTitle(),
+      LL.FullOnboarding.confirmNameContent({ firstName, lastName }),
+      [
+        { text: LL.common.cancel(), onPress: () => {} },
+        {
+          text: LL.common.yes(),
+          onPress: onfidoStart,
+        },
+      ],
+    )
+  }
+
+  const onfidoStart = async () => {
+    if (!accountId) {
+      console.log("no account id")
+      return
+    }
+
+    const res = await onboardingFlowStart({
+      variables: { input: { accountId, firstName, lastName } },
+    })
+
+    const workflowRunId = res.data?.onboardingFlowStart?.workflowRunId
+    if (!workflowRunId) {
+      Alert.alert("no workflowRunId")
+      return
+    }
+
+    const tokenAndroid = res.data?.onboardingFlowStart?.tokenAndroid
+    const tokenIos = res.data?.onboardingFlowStart?.tokenIos
+
+    const sdkToken = isIos ? tokenIos : tokenAndroid
+
+    if (!sdkToken) {
+      Alert.alert("no sdkToken")
+      return
+    }
+
+    try {
+      /* eslint @typescript-eslint/ban-ts-comment: "off" */
+      // @ts-expect-error
+      const res = await Onfido.start({
+        sdkToken,
+        theme: OnfidoTheme.AUTOMATIC,
+        workflowRunId,
+      })
+
+      Alert.alert(LL.FullOnboarding.success())
+      console.log(res, "success")
+    } catch (err) {
+      Alert.alert(LL.FullOnboarding.error())
+      console.log(err, "error")
+    }
+  }
+
+  if (currentLevel === AccountLevel.Two) {
+    return (
+      <Screen
+        preset="scroll"
+        keyboardShouldPersistTaps="handled"
+        keyboardOffset="navigationHeader"
+      >
+        <Text style={styles.textHeader}>
+          {LL.FullOnboarding.accountVerifiedAlready()}
+        </Text>
+      </Screen>
+    )
+  }
+
+  return (
+    <Screen
+      preset="scroll"
+      keyboardShouldPersistTaps="handled"
+      keyboardOffset="navigationHeader"
+    >
+      <View style={styles.screenContainer}>
+        <Text style={styles.textHeader}>{LL.FullOnboarding.requirements()}</Text>
+        <Input
+          placeholder={LL.FullOnboarding.firstName()}
+          value={firstName}
+          onChangeText={(text) => setFirstName(text)}
+        />
+        <Input
+          placeholder={LL.FullOnboarding.lastName()}
+          value={lastName}
+          onChangeText={(text) => setLastName(text)}
+        />
+        <GaloyPrimaryButton
+          onPress={confirmNames}
+          title={LL.common.next()}
+          disabled={!firstName || !lastName}
+        />
+      </View>
+    </Screen>
+  )
+}
+
+const useStyles = makeStyles(() => ({
+  view: {
+    marginHorizontal: 20,
+    padding: 20,
+    borderRadius: 20,
+  },
+
+  textHeader: {
+    fontSize: 18,
+    marginBottom: 24,
```

Comment:
Instead of using margin everywhere. The top level container can use the field `rowGap` which will space out all items in the container by that much.